docs/guides/backends.md

```diff
@@ -334,11 +334,11 @@ spec:
 
 
 ### OpenStack Swift
-Stash supports [OpenStack Swift as backend](https://restic.readthedocs.io/en/stable/030_preparing_a_new_repo.html#openstack-swift). To configure this backend, following secret keys are needed:
+Stash supports [OpenStack Swift as backend](https://backup.readthedocs.io/en/stable/030_preparing_a_new_repo.html#openstack-swift). To configure this backend, following secret keys are needed:
```

Comment:
Use: https://restic.readthedocs.io/en/stable/030_preparing_a_new_repo.html#openstack-swift instead of https://backup.readthedocs.io/en/stable/030_preparing_a_new_repo.html#openstack-swift